guppy/src/androidTest/java/com/jnj/guppy/ExampleInstrumentedTest.java

```diff
@@ -1,11 +1,13 @@
 package com.jnj.guppy;
 
 import android.content.Context;
-import androidx.test.InstrumentationRegistry;
```

Comment:
I run optimize imports in android studio. I missed that I touched example file.

Reply:
All good!